### PR TITLE
Add initial guide for 2019 election

### PR DESCRIPTION
**Pull request by castrojo**
This also updates the OWNERs to reflect the new election officers. 

Signed-off-by: Jorge O. Castro <jorgec@vmware.com>

/sig contributor-experience
/committee steering
/assign @parispittman @mrbobbytables @idvortetskyi @bgrant0607

### CODE REVIEW

**Comment by mrbobbytables** [comment]
/hold
Just for general ack.

**Comment by dims** [comment]
👍

**Comment by dims** [comment]
👍

File: events/elections/2019/README.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+## Background
+
+This election will shape the future of Kubernetes as a community and project.
+While SIGs and WGs help shape the technical direction of the project, the
+[Steering Committee Charter] covers the health of the project and community
+as a whole. Some examples of responsibilities to consider as you are deciding
+whether to run or who to vote for:
+
+- Define, evolve, and defend the vision, values, mission, and scope of the
```
Comment (bgrant0607) [inline events/elections/2019/README.md]: Please update this to reflect the updated bullet(s) in the charter:
https://github.com/kubernetes/steering/blob/master/charter.md#direct-responsibilities-of-the-steering-committee

File: events/elections/2019/README.md [hunk 2]
```diff
@@ -11,17 +11,22 @@ member will serve a two (2) year term.
...
-  contributors become committers/maintainers, approvers, reviewers, members,
-  etc.
-- Charter and refine policy for defining new community groups (including
-  Special Interest Groups, Working Groups, and Committees), and establish
-  transparency and accountability policies for such groups
+* Through the chartering review process, delegate ownership of, responsibility for
+  and authority over areas of the project to specific entities.
+* Define, evolve, and defend the non-technical vision / mission and the values
+  of the project.
+* Charter and refine policy for defining new community groups.
```
Comment (tpepper) [inline events/elections/2019/README.md]: errant trailing period

File: events/elections/2019/README.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+Please refer to the [Steering Committee Election Charter] for:
+
+- [eligibility for candidacy]
+- [eligibility for voting] and [eligible voters] list (not in charter)
+
+### Schedule
+
+| Date         | Event                    |
+| ------------ | ------------------------ |
+| August 23    | Announcement of Election |
```
Comment (bgrant0607) [inline events/elections/2019/README.md]: Reminder to update the dates and times

Comment (bgrant0607) [reply events/elections/2019/README.md]: 21 August - Announce Election - first thing in the morning EST, this is
just sending an email
11 Sept - All bios and voting exemptions due - 5pm PST
18 Sept - Voting begins - "Morning of" to account for CIVS setup
2 October - Voting ends - 5pm PST
3 October - Results published on blog/community meeting - Normal
community meeting time, blog published right after.

File: events/elections/2019/README.md (hunk 2 above)
Comment (bgrant0607) [reply events/elections/2019/README.md]: Ah, yes, that's the middle of the sentence

File: events/elections/2019/README.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+- [Eligibility for candidacy]
+- [Eligibility for voting] and [Eligible voters] list (not in charter)
+
+### Schedule
+
+| Date         | Event                    |
+| ------------ | ------------------------ |
+| August 21    | Announcement of Election |
+| September 11 | All candidate bios and voting exception forms due by 0000 UTC (5pm PST) |
+| September 28 | Election Begins via email ballots |
```
Comment (bgrant0607) [inline events/elections/2019/README.md]: This should be Sept 18

File: events/elections/2019/README.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+- [Eligibility for voting] and [Eligible voters] list (not in charter)
+
+### Schedule
+
+| Date         | Event                    |
+| ------------ | ------------------------ |
+| August 21    | Announcement of Election |
+| September 11 | All candidate bios and voting exception forms due by 0000 UTC (5pm PST) |
+| September 18 | Election Begins via email ballots |
+| October 2    | Election Closes byy 0000 UTC (5pm PST) |
```
Comment (mrbobbytables) [inline events/elections/2019/README.md]:
```suggestion
| October 2    | Election Closes by 0000 UTC (5pm PST) |
```

Super tiny nit :x

File: events/elections/README.md [hunk 6]
```diff
@@ -79,7 +80,7 @@ eligibility for voting, eligibility for candidacy, maximal representation, etc.
 
 - Must be [eligible to vote][eligible to vote]
 - Cannot be running for office in the current election
-- Cannot be a current member of the steering committee
+- Cannot be a current member of the steering committee that is up for re-election or whose term extends beyond the election period
```
Comment (bgrant0607) [inline events/elections/README.md]: s/that is up for re-election/that is a candidate in the election/

**Comment by bgrant0607** [comment]
Thanks!
/hold cancel
/lgtm
/approve

**Pull request merged**